client/blocks/site-preview/index.jsx

```diff
@@ -1,46 +1,119 @@
 /** @format */
-
 /**
  * External dependencies
  */
-
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
-import { noop } from 'lodash';
+import debugFactory from 'debug';
 
 /**
  * Internal dependencies
  */
-import DesignPreview from 'my-sites/design-preview';
-import UrlPreview from 'blocks/url-preview';
+import { closePreview } from 'state/ui/preview/actions';
+import { getPreviewSite, getPreviewSiteId, getPreviewUrl } from 'state/ui/preview/selectors';
+import { getSiteOption, getSiteSlug } from 'state/sites/selectors';
 import { getCurrentLayoutFocus } from 'state/ui/layout-focus/selectors';
-import { getCurrentPreviewType } from 'state/ui/preview/selectors';
+import { addQueryArgs } from 'lib/route';
+import isDomainOnlySite from 'state/selectors/is-domain-only-site';
+import WebPreview from 'components/web-preview';
+
+const debug = debugFactory( 'calypso:design-preview' );
```

Comment:
This should probably be changed to `calypso:site-preview`

Reply:
Changed to `calypso:site-preview`.

Thanks @sirbrillig for looking at the PR. All issues are solved now and it's ready for another look.